app/src/test/java/com/github/se/signify/model/user/UserViewModelTest.kt

```diff
@@ -87,4 +88,16 @@ class UserViewModelTest {
     userViewModel.removeFriend(currentUserId, friendUserId)
     verify(userRepository).removeFriend(eq(currentUserId), eq(friendUserId), any(), any())
   }
+
```

Comment:
The two added tests check if remove & add a challenge is being called in the user repo bu using a mocked model.
Simple and efficient good job